<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>861</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string> Animal</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Afficher">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AFFICHER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>AGE : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_race">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>140</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sexe">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Supprimer">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>ID : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_race">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>180</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>RACE : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_prix">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>PRIX : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_sexe">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>SEXE : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="Modifier">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="a">
     <attribute name="title">
      <string>suivi</string>
     </attribute>
     <widget class="QPushButton" name="Afficher_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AFFICHER</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>351</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_prix_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>TAILLE : </string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_fiche">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>150</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom_2">
      <property name="geometry">
       <rect>
        <x>426</x>
        <y>123</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NOMBRE_CONSULTATION : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nbr_fiche">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_taille_fiche">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Supprimer_fiche">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>POID : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_Fiche_animal">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Modifier_fiche">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_poid_fiche">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_id_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>ID : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_FICHE">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_animal_fiche">
      <property name="geometry">
       <rect>
        <x>564</x>
        <y>187</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>150</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DATE vacccination</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom_4">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>180</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID ANIMAL : </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
